addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -68,6 +68,18 @@
 </template>
 
 <!-- COURSE -->
+<template name="Course Main" id="course_main" inherit_id="website_slides.course_main">
+    <xpath expr="//div[@id='home']" position="before">
+        <div t-if="channel.enroll == 'payment' and not channel.product_id.is_published"
+            class="alert alert-info" role="alert" groups="website_slides.group_website_slides_officer">
+            This course cannot be bought because its linked product
+            <a t-attf-href="/web#id=#{channel.product_id.id}&amp;view_type=form&amp;model=#{channel.product_id._name}"
```

Comment:
Please note that this empties the navbar of the app (because the menu_id is not provided).

Recently I used this kind of link:
`<a t-attf-href="/web#view_type=form&amp;model=chatbot.script&amp;id=#{chatbot_data['chatbot_script_id']}&amp;action=im_livechat.chatbot_script_action">`

Maybe you could give it a try?
Would have to replace the action by the main product action of course, looks like it's "product_product_action", to double-check.